translations/ui-quick-marc/en.json

```diff
@@ -760,6 +760,9 @@
   "record.save.error.notFound": "This record has been deleted by another user. You can no longer edit this record, hit the cancel button to return to the previous page.",
   "record.save.error.generic": "Record not saved: Communication problem with server. Please try again.",
   "record.save.error.derive": "Instance cannot be updated. Deprecated instance version.",
+  "record.save.error.warn": "<b>Please scroll to view the entire record. Resolve errors as needed and save to revalidate the record.</b>{breakingLine}Warn errors: <b>{warnCount}</b>",
+  "record.save.error.fail": "<b>Please scroll to view the entire record. Resolve errors as needed and save to revalidate the record.</b>{breakingLine}Fail errors: <b>{failCount}</b>{breakingLine}Record cannot be saved with a fail error.",
+  "record.save.error.failAndWarn": "<b>Please scroll to view the entire record. Resolve errors as needed and save to revalidate the record.</b>{breakingLine}Warn errors: <b>{warnCount}</b>{breakingLine}Fail errors: <b>{failCount}</b>{breakingLine}Record cannot be saved with a fail error.",
```

Comment:
* `s/warn errors/warnings/`: warnings are notifications that something is not quite right, but validation will still succeed. A record with warnings can be saved.
* `s/fail errors/errors/`: an error means validation failed. A record with errors cannot be saved. Alternative to "errors": "failures", but I think "errors" is more common. 

Maybe use language like, "Resolve issues as needed..." since "issues" can cover both "warnings" and "errors"?